src/styles/_reboot.scss

```diff
@@ -38,8 +38,14 @@
 html {
   font-family: sans-serif; // 2
   line-height: 1.15; // 3
+  -webkit-text-size-adjust: none; // 4
```

Comment:
vendor prefix shoukld be done by the post css which we have. please try without and ispoect the css